app/templates/min_funding_amount.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+
+{%- from 'govuk_frontend_jinja/components/input/macro.html' import govukInput -%}
+
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+
+{% block content %}
+
+<div>
+  <h3 class="govuk-heading-m">How much money does your project need?</h3>
+</div>
+<form method="POST">
+  {{ form.csrf_token }}
+{{
+  govukInput({
+    "label": {"text": "How much money do you need?"},
+    "name": form.money_field.id,
+    "label": {"text": "Enter your answer using numbers only"},
+    "prefix": {"text": "£"},
+    "classes": "govuk-input--width-20",
+    "type" : "number"
```

Comment:
Where do we find Input options?

Reply:
The components and their options can be found here https://govuk-frontend-jinja.herokuapp.com